### PR TITLE
add methods for getting all product combinations

### PR DESCRIPTION
**Pull request by moran711**

### CODE REVIEW

File: modules/product/product.service.js [hunk 1]
```diff
@@ -202,6 +204,19 @@ class ProductsService {
     }
   }
 
+  async getProductDetailsCombinations() {
```
Comment (Kiyooooko) [inline modules/product/product.service.js:207]: something wrong with patterns
Screenshot 2021-01-30 at 14 10 49

Comment (moran711) [reply modules/product/product.service.js:207]: solved

**Review by Kiyooooko — DISMISSED** [review]

**Comment by Kiyooooko** [comment]
🔥 🚀

**Comment by Kiyooooko** [comment]
🔥

**Review by Kiyooooko — DISMISSED** [review]

**Comment by moran711** [comment]
> 🔥
🤜🏻🤛🏽

**Review by AndriyLymych — APPROVED** [review]

**Review by Kiyooooko — APPROVED** [review]

**Review by AndriyLymych — DISMISSED** [review]

**Comment by AndriyLymych** [comment]
🔥 🚀🔥 🚀🔥 🚀

**Review by NikitaDenysenko — APPROVED** [review]

**Review by AndriyLymych — APPROVED** [review]

**Review by Kiyooooko — APPROVED** [review]

**Pull request merged**